pkg/driverbuilder/builder/image.go

```diff
@@ -19,9 +19,24 @@ type Image struct {
 	Name       string
 }
 
+type ImagesLister interface {
+	LoadImages()
+}
+
+type FileImagesLister struct {
+	file     *os.File
+	Build    *Build
+	FilePath string
+}
+
+type RepoImagesLister struct {
+	Repo  string
+	Build *Build
```

Comment:
I'd avoid storing the `Build` here! Instead, i'd pass it to the `ListImages` methods, as a param.
I think it is better for these methods to be stateless: just return a list of `Image`s.
Moreover, given that `regs` are now just used by `RepoImagesLister`, i think they can just be declared inside the `ListImages` method.

Reply:
I think they can also be global const variables in image.go, and then referenced in each `RepoImagesLister`; i don't like adding yet another non-build related field to `Build` struct.
Given it is only used by `RepoImagesLister`, i think it should live there!